Provision/base.sql

```diff
@@ -65,7 +65,8 @@ CREATE TABLE IF NOT EXISTS summoners
 CREATE TABLE IF NOT EXISTS matches
 (
     id INT NOT NULL AUTO_INCREMENT,
-    role ENUM('Top', 'Jungle', 'Middle', 'Bottom', 'Support') not null,
+    match_id INT NOT NULL,
+    lane ENUM('Top', 'Jungle', 'Middle', 'Bottom', 'Support') not null,
```

Comment:
The league of legends api provides us both lane and role. Do we have any interest in combining these? Solo Top, Duo Bottom, etc?